intercom-java/src/main/java/io/intercom/api/Conversation.java

```diff
@@ -214,6 +214,9 @@ private static boolean isAdminQuery(Map<String, String> params) {
 
     @JsonProperty("conversation_message")
     private ConversationMessage conversationMessage;
+    
+    @JsonProperty("customer_first_reply")
+    private CustomerFirstReply customerFirstReply;
```

Comment:
I think you need to add a public getter for `customerFirstReply` here https://github.com/intercom/intercom-java/blob/3e5302cacc7682544d49f41ad9bb0b19f4983137/intercom-java/src/main/java/io/intercom/api/Conversation.java#L300

Could you also add `customerFirstReply` to `hashCode` and `equals`?